src/libsync/syncfilestatustracker.cpp

```diff
@@ -157,6 +162,20 @@ SyncFileStatus SyncFileStatusTracker::fileStatus(const QString& systemFileName)
     return SyncFileStatus();
 }
 
+void SyncFileStatusTracker::slotPathTouched(const QString& systemFileName)
+{
+    QString fileName = systemFileName.normalized(QString::NormalizationForm_C);
+
+    QString folderPath = _syncEngine->localPath();
+    if (!fileName.startsWith(folderPath))
```

Comment:
This should normally always be the case since the folderwatcher watches only files within that folder, so this could be a Q_ASSERT